devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -0,0 +1,397 @@
+/**
+ *      Min Smart Plug v1.0.4
+ *
+ *  	Models: MINOSTON (MP21Z)
+ *
+ *  Author:
+ *   winnie (sky-nie)
+ *
+ *	Documentation:
+ *
+ *  Changelog:
+ *
+ *    1.0.4 (07/13/2021)
+ *      - Syntax format compliance adjustment
+ *      - delete dummy code
+ *
+ *    1.0.3 (07/12/2021)
+ *    1.0.2 (07/07/2021)
+ *      - delete dummy code
+ *
+ *    1.0.1 (03/17/2021)
+ *      - Simplify the code, delete dummy code
+ *
+ *    1.0.0 (03/11/2021)
+ *      - Initial Release
+ *
+ * Reference：
+ *  https://github.com/krlaframboise/SmartThings/blob/master/devicetypes/krlaframboise/eva-logik-in-wall-smart-switch.src/eva-logik-in-wall-smart-switch.groovy
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+metadata {
+    definition (name: "Min Smart Plug",namespace: "sky-nie",author: "winnie",mnmn: "SmartThings", vid:"generic-switch", ocfDeviceType: "oic.d.smartplug" ) {
+        capability "Actuator"
+        capability "Sensor"
+        capability "Switch"
+        capability "Light"
+        capability "Configuration"
+        capability "Refresh"
+
```

Comment:
as I said earlier, `capability "Health Check"` SHOULDN'T be removed.

Reply:
sorry, I forget it.

Reply:
Sorry, I read it wrong